pkg/util/validation/validate_test.go

```diff
@@ -493,3 +531,14 @@ func TestMaxNativeHistorgramBuckets(t *testing.T) {
 			cortex_discarded_samples_total{group="group-1",reason="max_native_histogram_buckets",user="user-1"} 8
 	`), "cortex_discarded_samples_total"))
 }
+
+func tooManyLabelsArgs(series []mimirpb.LabelAdapter, limit int) []any {
+	metric := mimirpb.FromLabelAdaptersToMetric(series).String()
+	ellipsis := ""
+
+	if utf8.RuneCountInString(metric) > 200 {
+		ellipsis = "\u2026"
+	}
+
+	return []any{len(series), limit, metric, ellipsis}
```

Comment:
Shouldn't we shorten the metric if it's too long?

Reply:
This is the same implementation we are having on master right now ([here](https://github.com/grafana/mimir/blob/main/pkg/util/validation/errors.go#L103-L114))

Reply:
Hum, okay, I'd say that's a bug.